cpp/turbodbc/Test/tests/make_description_of_description_test.cpp

```diff
@@ -225,6 +225,40 @@ TEST(MakeDescriptionOfDescriptionTest, VarcharFieldsCanBeLimited)
     EXPECT_EQ(limit + 1, limited->element_size());
 }
 
+TEST(MakeDescriptionOfDescriptionTest, VarcharFieldsModifiedMemoryAllocation)
+{
+    std::size_t const reported_size = 10;
+    std::size_t const allocated_size = 40;
+    cpp_odbc::column_description column_description = {name, SQL_VARCHAR, reported_size, 0, supports_null_values};
+
+    turbodbc::options options;
+    options.fix_unicode_allocation = false;
```

Comment:
Since you recycle the options struct, I think it is best to remove the names `allocation` and `modified_allocation`:
```
turbodbc::options options;

options.fix_unicode_allocation = false;
EXPECT_EQ(reported_size + 1, make_description(column_description, options)->element_size());

options.fix_unicode_allocation = true;
EXPECT_EQ(size_with_extra_capacity + 1, make_description(column_description, options)->element_size());
```